utils.py

```
@@ -0,0 +1,35 @@
+"""
+Utility functions for the RPEC RAG application.
+"""
+import streamlit as st
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)